fastchat/model/model_adapter.py

```diff
@@ -1547,6 +1564,7 @@ def get_default_conv_template(self, model_path: str) -> Conversation:
 
 # Note: the registration order matters.
 # The one registered earlier has a higher matching priority.
+register_model_adapter(ReaLMAdapter)
```

Comment:
move this to the end of the list

Reply:
> move this to the end of the list

done it, thx!!

Reply:
> move this to the end of the list

We also infer and evaluate our model by ourselves, do your group need the JSON file?

Reply:
do you mean mt-bench JSON files? please share it with a public link and we may add it to our database later.

Reply:
> do you mean mt-bench JSON files? please share it with a public link and we may add it to our database later.

Yeah. it was uploaded to our repo:
[https://github.com/FreedomIntelligence/ReaLM/tree/main/data/mt-bench](mt-bench-results)

Reply:
> do you mean mt-bench JSON files? please share it with a public link and we may add it to our database later.

hi, bro. can I know when will your group add our model to mt-bench?
although I want to iterate it for better performance, my boss pushed me to add it